docs/plugins/playlist.rst

```diff
@@ -24,6 +24,16 @@ name::
 
     $ beet ls playlist:anotherplaylist
 
+A playlist query will use the paths found in the playlist file to match items
+in the beets library. Since `playlist:` submits a regular beets query like any
```

Comment:
Maybe consider linking to the [query page](https://github.com/beetbox/beets/blob/master/docs/reference/query.rst)? Then we probably don't need the "e.g." parenthetical to explain more.